### PR TITLE
Always render adblock test element.

### PR DESCRIPTION
**Pull request by dwick**
We want to test for adblock on every page.  Google tag manager
should continue to be excluded for users with ads disable though.

👓 @schwers 

related to https://github.com/reddit/reddit-mobile/pull/654/files

### CODE REVIEW

File: src/views/layouts/BodyLayout.jsx [hunk 1]
```diff
@@ -73,32 +73,35 @@ class BodyLayout extends BasePage {
...
       width: '1px',
       position: 'absolute',
       left: '-1000%',
     };
+    const adCode = [
+      <div
+        id="adblock-test"
+        className={ adblockTestClassName }
+        style={ adblockStyles }
+      />
```
Comment (schwers) [inline src/views/layouts/BodyLayout.jsx]: Linter wants a trailing comma here

File: src/views/layouts/BodyLayout.jsx [hunk 2]
```diff
@@ -73,32 +73,35 @@ class BodyLayout extends BasePage {
...
     const { mediaDomain, googleTagManagerId, adblockTestClassName } = app.config;
     const adblockStyles = {
       height: '1px',
       width: '1px',
       position: 'absolute',
       left: '-1000%',
     };
+    const adCode = [
+      <div
+        id="adblock-test"
```
Comment (schwers) [inline src/views/layouts/BodyLayout.jsx]: Can you add a key property here? React complains in the console if you render an array and all of the elements don't have unique keys.

File: src/views/layouts/BodyLayout.jsx [hunk 3]
```diff
@@ -73,32 +73,35 @@ class BodyLayout extends BasePage {
...
-      <div>
-        <div
-          id="adblock-test"
-          className={ adblockTestClassName }
-          style={ adblockStyles }
-        />
+    if (adsEnabled && googleTagManagerId) {
+      adCode.push(
         <GoogleTagManager
           nonce={ ctx.csrf }
```
Comment (schwers) [inline src/views/layouts/BodyLayout.jsx]: Same comment about keys since this is rendered in an array now

**Comment by schwers** [comment]
Thanks for doing this. I just have a few lint'ing and react logging related nits, looks good overall 💅

**Comment by dwick** [comment]
I really need to set my hooks up.

**Comment by dwick** [comment]
💇

**Comment by schwers** [comment]
:fish:

**Pull request merged**